### ISSUE TITLE
aria-label should be defined for all blocks and values

### ISSUE THREAD

**Issue by schanzer** [bootstrapworld/codemirror-blocks#10]
For accessibility purposes, all blocks should have their aria-label attribute set.

From "Accessibility" in the spec:
* Symbols, Numbers and Boolean values should be set to themselves
* String values should be set to “string ”+the contents of the string
* Expression blocks should be set to <function name> + “expression,” + <num arguments> (e.g. - “star expression, three arguments”)

**Comment by pcardune** [comment]
For expressions, do you mean

    (star "red" 10 10)

should become "star expression, 3 arguments"

**Comment by schanzer** [comment]
Yep

**Comment by pcardune** [comment]
Oh, what about and/or expressions?

**Comment by schanzer** [comment]
For the novice learners who are using this tool, I'm comfortable saying and/or expressions are the same as any other expression (indeed, they use the same block appearance!). 

(and true (> 3 4)) would read as "and expression, 2 arguments"
